modules/integration/tests/src/test/java/org/wso2/appmanager/integration/utils/HttpUtil.java

```diff
@@ -188,80 +95,31 @@ public static HttpResponse doDelete(URL endpoint,
      * @return httpResponse HttpResponse.
      * @throws Exception on errors.
      */
-    public static HttpResponse doPost(URL endpoint, String postBody,
-                                      Map<String, String> headers) throws Exception {
+    public static HttpResponse doPost(URL endpoint, String postBody, Map<String, String> headers) throws IOException {
         HttpURLConnection urlConnection = null;
+        Writer writer = null;
+        OutputStream out = null;
```

Comment:
Fixed in https://github.com/wso2/product-app-manager/pull/357/commits/d5adfb28b2122712d0da0911c0d4921cf5bd98d9